vigranumpy/src/core/axistags.cxx

```diff
@@ -564,6 +564,8 @@ void defineAxisTags()
          "    :type typeFlags: :class:`~vigra.AxisType`\n"
          "    :param resolution: the resolution (step size) of the axis\n"
          "                       (e.g. 0.0 means 'unknown')\n"
+         "    :param unit: the unit of measurement corresponding to resolution of the axis\n"
+         "                       (can be left undefined)\n"
```

Comment:
the docstring needs more updating. 

1. in line 558, the docstring does not include the unit (it should!)
1. The position of the unit arg should be after the description (logically it belongs to resolution, but the order of arguments matters. `type = vigra.AxisInfo("x", vigra.AxisType.Space, 0, "description")` still works as expected. Reading the docstring I'd have expected to set `unit` to the text `description`.
2. Further down, there are a lot of descriptions of the factories, these need updating with unit, too.